src/components/Book.jsx

```diff
@@ -6,17 +6,20 @@ export default function Book({ book }) {
   return (
     <div className="flex items-center justify-between px-2 py-4 shadow-md">
       <div className="flex items-start justify-start flex-col">
-        <h2 className="text-xl font-semibold">
+        <span className="text-sm text-gray-500">
+          {book.category}
+        </span>
+        <h2 className="text-2xl font-semibold">
           {book.title}
         </h2>
-        <h4 className="text-md italic text-grey-500">
+        <span className="text-sm italic text-blue-500">
           {book.author}
-        </h4>
+        </span>
       </div>
       <button
         type="button"
         onClick={() => dispatch(delBook(book.id))}
-        className="bg-red-500 hover:bg-red-400 py-1 px-2 text-white rounded-lg shadow-sm"
+        className="bg-blue-500 hover:bg-blue-400 py-1 px-2 text-white rounded-lg shadow-sm"
       >
         Delete Book
       </button>
```

Comment:
- [ ] Let's try to stick to the project requirements as this is setting us up for real world projects in the future when we get a real job. Kindly style your app to have all sections and placeholders even if they're not doing anything in the app at the moment but for project guideline and requirement, let's work according to how we're required to style the app. 👍🏽
- [ ] Currently your app does not mact the original design given, hence, you are to match the original design in order for your project to be complete. Please work on it 👍🏽

**Your App**

<img width="1642" alt="Screenshot 2023-05-12 at 6 13 31 PM" src="https://github.com/kimitaWanjohi/bookstore/assets/50941074/39be183b-676b-4c4e-98ee-50f2312f356e">

**The Original Design**

<img width="1031" alt="Screenshot 2023-05-12 at 6 11 24 PM" src="https://github.com/kimitaWanjohi/bookstore/assets/50941074/13dc08f1-f74a-4ad4-bc18-1a01670c8e65">